src/main/kotlin/com/avail/interpreter/levelTwo/L1InstructionStepper.kt

```diff
@@ -1009,10 +1009,8 @@ class L1InstructionStepper constructor(val interpreter: Interpreter)
 		val functionToCall =
 			interpreter.runtime().invalidMessageSendFunction()
 		interpreter.isReifying = true
-		return StackReifier(
-			true,
-			reificationForFailedLookupStat
-		) {
+		return StackReifier(true, reificationForFailedLookupStat)
+		{
```

Comment:
There are certain things it can't do, but it can definitely do this. It can't handle things like foreach:

```
arrayListOf(1, 2, 3).forEach 
{ 
...			
}
```
But it can generally handle:
```
foo.someFunc(input1, input2)
{
   // input 3 is a lambda
}
```
I definitely prefer the style I put forth as well.